Submitted By: BLFS Book <blfs-book@linuxfromscratch.org>
Date: 2003-10-04
Initial Package Version: 4.44
Origin: BLFS Book
Description: Uses FHS compilant paths.
diff -ur pine4.56/README pine4.56.fhs/README
--- pine4.56/README	2003-01-09 12:44:24.000000000 -0600
+++ pine4.56.fhs/README	2003-08-03 09:28:56.000000000 -0500
@@ -277,8 +277,8 @@
  pine   The Pine mailer. Once compiled this should work just fine on
         your system with no other files than this binary, and no
         modifications to your system. Optionally you may create two
-        configuration files, /usr/local/lib/pine.conf and 
-        /usr/local/lib/pine.info. See the documentation for details.
+        configuration files, /etc/pine.conf and 
+        /etc/pine.info. See the documentation for details.
  
  pico   The standalone editor similar to the Pine message composer.
         This is a very simple straight forward text editor.
@@ -304,7 +304,7 @@
 	be they local or remote.
 
 In general you should be able to just copy the Pine and Pico binaries
-to the place you keep your other local binaries. /usr/local/bin is a
+to the place you keep your other local binaries. /usr/bin is a
 likely place.
   
 -----------------------------------------------------------------------
diff -ur pine4.56/doc/pine.1 pine4.56.fhs/doc/pine.1
--- pine4.56/doc/pine.1	2003-05-29 14:39:31.000000000 -0500
+++ pine4.56.fhs/doc/pine.1	2003-08-03 09:28:56.000000000 -0500
@@ -333,11 +333,11 @@
 .br
 /etc/mime.types	System-wide file ext. to MIME type mapping
 .br
-/usr/local/lib/pine.info	Local pointer to system administrator.
+/etc/pine.info	Local pointer to system administrator.
 .br
-/usr/local/lib/pine.conf	System-wide configuration file.
+/etc/pine.conf	System-wide configuration file.
 .br
-/usr/local/lib/pine.conf.fixed	 Non-overridable configuration file.
+/etc/pine.conf.fixed	 Non-overridable configuration file.
 .br
 /tmp/.\\usr\\spool\\mail\\xxxx	Per-folder mailbox lock files.
 .br
diff -ur pine4.56/doc/tech-notes/background.html pine4.56.fhs/doc/tech-notes/background.html
--- pine4.56/doc/tech-notes/background.html	2003-03-26 13:52:51.000000000 -0600
+++ pine4.56.fhs/doc/tech-notes/background.html	2003-08-03 09:28:56.000000000 -0500
@@ -174,9 +174,9 @@
 <OL>
 
 <LI><EM>sendmail-path</EM> in
-<CODE>/usr/local/lib/pine.conf.fixed</CODE>
+<CODE>/etc/pine.conf.fixed</CODE>
 
-<LI><EM>smtp-server</EM> in <CODE>/usr/local/pine.conf.fixed</CODE>
+<LI><EM>smtp-server</EM> in <CODE>/etc/pine.conf.fixed</CODE>
 
 <LI><EM>sendmail-path</EM> specified on the command line. 
 
@@ -186,9 +186,9 @@
 
 <LI><EM>smtp-server</EM> in the user's <CODE>.pinerc</CODE> file. 
 
-<LI><EM>sendmail-path</EM> in <CODE>/usr/local/lib/pine.conf</CODE>
+<LI><EM>sendmail-path</EM> in <CODE>/etc/pine.conf</CODE>
 
-<LI><EM>smtp-server</EM> in <CODE>/usr/local/pine.conf</CODE>
+<LI><EM>smtp-server</EM> in <CODE>/etc/pine.conf</CODE>
 
 <LI><CODE>DF_SENDMAIL_PATH</CODE> defined at compile time.
 
diff -ur pine4.56/doc/tech-notes/cmd-line.html pine4.56.fhs/doc/tech-notes/cmd-line.html
--- pine4.56/doc/tech-notes/cmd-line.html	2003-04-16 12:51:00.000000000 -0500
+++ pine4.56.fhs/doc/tech-notes/cmd-line.html	2003-08-03 09:28:56.000000000 -0500
@@ -87,7 +87,7 @@
 file, execute
 
 <PRE><CODE>
-		pine -conf > /usr/local/lib/pine.conf
+		pine -conf > /etc/pine.conf
 </CODE></PRE>
 <P>
 
@@ -95,7 +95,7 @@
 system configuration file, execute
 
 <PRE><CODE>
-		pine -P old-pine.conf -conf > /usr/local/lib/pine.conf
+		pine -P old-pine.conf -conf > /etc/pine.conf
 </CODE></PRE>
 <P>
 A system configuration file is not required.
@@ -264,7 +264,7 @@
 <DT> -P <EM>pinerc</EM>
 
 <DD> Uses the named file as the system wide configuration file instead of
-<EM>/usr/local/lib/pine.conf</EM> on UNIX, or nothing on <EM>PC-Pine</EM>.
+<EM>/etc/pine.conf</EM> on UNIX, or nothing on <EM>PC-Pine</EM>.
 Pinerc may be either a local file or a remote configuration folder.
 <P>
 
Only in pine4.56.fhs/doc/tech-notes: cmd-line.html~
diff -ur pine4.56/doc/tech-notes/config-notes.html pine4.56.fhs/doc/tech-notes/config-notes.html
--- pine4.56/doc/tech-notes/config-notes.html	2003-04-24 17:31:27.000000000 -0500
+++ pine4.56.fhs/doc/tech-notes/config-notes.html	2003-08-03 09:28:56.000000000 -0500
@@ -883,10 +883,10 @@
 </OL>
 <P>
 The fixed configuration file is normally
-<CODE>/usr/local/lib/pine.conf.fixed</CODE>.
+<CODE>/etc/pine.conf.fixed</CODE>.
 <P>
 The system-wide configuration file is normally
-<CODE>/usr/local/lib/pine.conf</CODE> for Unix <EM>Pine</EM> and is normally not
+<CODE>/etc/pine.conf</CODE> for Unix <EM>Pine</EM> and is normally not
 set for <EM>PC-Pine</EM>.
 For <EM>PC-Pine</EM>, if the environment variable <EM>$PINECONF</EM> is set, that
 is used for the system-wide configuration.
@@ -1187,8 +1187,8 @@
 
 <LI> An entry in <CODE>/etc/inetd.conf</CODE> for the alternate service.
 
-<LI> An entry in <CODE>/usr/local/lib/pine.conf</CODE>,
-<CODE>/usr/local/lib/pine.conf.fixed</CODE> or <CODE>~/.pinerc</CODE>. 
+<LI> An entry in <CODE>/etc/pine.conf</CODE>,
+<CODE>/etc/pine.conf.fixed</CODE> or <CODE>~/.pinerc</CODE>. 
 
 </UL>
 
Only in pine4.56.fhs/doc/tech-notes: config-notes.html~
diff -ur pine4.56/doc/tech-notes/config.html pine4.56.fhs/doc/tech-notes/config.html
--- pine4.56/doc/tech-notes/config.html	2003-05-09 12:20:33.000000000 -0500
+++ pine4.56.fhs/doc/tech-notes/config.html	2003-08-03 09:28:56.000000000 -0500
@@ -7,8 +7,8 @@
 configuration. In most cases, the compiled-in preferences will suit users
 and administrators just fine. When running <EM>Pine</EM> on a UNIX system, the
 default built-in configuration can be changed by setting variables in the
-system configuration files, <CODE>/usr/local/lib/pine.conf</CODE>
-or <CODE>/usr/local/lib/pine.conf.fixed</CODE>.
+system configuration files, <CODE>/etc/pine.conf</CODE>
+or <CODE>/etc/pine.conf.fixed</CODE>.
 (Actually, these files are whatever the definitions for
 <CODE>SYSTEM_PINERC</CODE> and <CODE>SYSTEM_PINERC_FIXED</CODE>
 in pine/osdep/os-xxx.h are set to.)
@@ -2468,7 +2468,7 @@
 alternate spell checker.
 If your Unix system has <EM>ispell</EM> it is probably reasonable to make
 it the default speller by configuring it as the default in the
-system configuration file, <CODE>/usr/local/lib/pine.conf</CODE>.
+system configuration file, <CODE>/etc/pine.conf</CODE>.
 
 <P>
 If this option is not set, then the system's <EM>spell</EM> command is used.
@@ -2501,7 +2501,7 @@
 <DT> <A NAME="ssh-path"><EM>ssh-path</EM></A>
 
 <DD> Sets the name of the command used to open a UNIX secure shell
-connection. The default is typically <CODE>/usr/local/bin/ssh</CODE>.
+connection. The default is typically <CODE>/usr/bin/ssh</CODE>.
 <P>
 
 <DT> <A NAME="standard-printer"><EM>standard-printer</EM></A>
Only in pine4.56.fhs/doc/tech-notes: config.html~
diff -ur pine4.56/doc/tech-notes/installation.html pine4.56.fhs/doc/tech-notes/installation.html
--- pine4.56/doc/tech-notes/installation.html	2003-02-05 21:04:26.000000000 -0600
+++ pine4.56.fhs/doc/tech-notes/installation.html	2003-08-03 09:28:56.000000000 -0500
@@ -218,7 +218,7 @@
 <DD> The name of the file which holds <EM>Pine</EM>
 configuration information for
 all users on the system.  Default on UNIX systems is
-<CODE>/usr/local/lib/pine.conf</CODE>.
+<CODE>/etc/pine.conf</CODE>.
 
 <DT> SYSTEM_PINERC_FIXED
 
@@ -226,7 +226,7 @@
 SYSTEM_PINERC, but only for variables that the administrator wants to keep
 fixed.  That is, users are not allowed to change variables that are
 specified in the FIXED file.  Default on UNIX systems is
-<CODE>/usr/local/lib/pine.conf.fixed</CODE>.
+<CODE>/etc/pine.conf.fixed</CODE>.
 
 </DL> <P>
 
@@ -529,9 +529,9 @@
 are no <STRONG>required</STRONG> auxiliary files.  <P>
 
 There are, however, three optional auxiliary files: 
-<CODE>/usr/local/lib/pine.info</CODE>,
-<CODE>/usr/local/lib/pine.conf</CODE>, and
-<CODE>/usr/local/lib/pine.conf.fixed</CODE>.  The file
+<CODE>/etc/pine.info</CODE>,
+<CODE>/etc/pine.conf</CODE>, and
+<CODE>/etc/pine.conf.fixed</CODE>.  The file
 <CODE>pine.info</CODE> contains text on how to get further help on the
 local system.  It is part of the help text for the
 main menu and should probably refer to the local help desk or the system
@@ -650,15 +650,15 @@
 
 <DL COMPACT>
 
-<DT> /usr/local/lib/pine.conf
+<DT> /etc/pine.conf
 
 <DD> Pine's global configuration file. 
 
-<DT> /usr/local/lib/pine.conf.fixed
+<DT> /etc/pine.conf.fixed
 
 <DD> Non-overridable global configuration file. 
 
-<DT> /usr/local/lib/pine.info
+<DT> /etc/pine.info
 
 <DD> Local pointer to system administrator. 
 
Only in pine4.56.fhs/doc/tech-notes: installation.html~
diff -ur pine4.56/doc/tech-notes/low-level.html pine4.56.fhs/doc/tech-notes/low-level.html
--- pine4.56/doc/tech-notes/low-level.html	2002-11-01 13:35:48.000000000 -0600
+++ pine4.56.fhs/doc/tech-notes/low-level.html	2003-08-03 09:28:56.000000000 -0500
@@ -1088,7 +1088,7 @@
 
 The second selection is the standard UNIX print command.  The default is
 <EM>lpr</EM>, but it can be changed on a system basis to anything so
-desired in <CODE>/usr/local/lib/pine.conf</CODE>. <P>
+desired in <CODE>/etc/pine.conf</CODE>. <P>
 
 The third selection is
 the user's personal choice for a UNIX print command.  The text to be
Only in pine4.56.fhs/doc/tech-notes: low-level.html~
diff -ur pine4.56/doc/tech-notes/porting.html pine4.56.fhs/doc/tech-notes/porting.html
--- pine4.56/doc/tech-notes/porting.html	1998-09-16 18:09:04.000000000 -0500
+++ pine4.56.fhs/doc/tech-notes/porting.html	2003-08-03 09:28:56.000000000 -0500
@@ -307,11 +307,11 @@
 <DT>
  ___
 <DD>
-Reading of <CODE>/usr/local/lib/pine.conf</CODE> 
+Reading of <CODE>/etc/pine.conf</CODE> 
 <DT>
  ___
 <DD>
-Fixing variables and features in <CODE>/usr/local/lib/pine.conf.fixed</CODE> 
+Fixing variables and features in <CODE>/etc/pine.conf.fixed</CODE> 
 <DT>
  ___
 <DD>
diff -ur pine4.56/doc/tech-notes.txt pine4.56.fhs/doc/tech-notes.txt
--- pine4.56/doc/tech-notes.txt	2003-05-29 14:41:12.000000000 -0500
+++ pine4.56.fhs/doc/tech-notes.txt	2003-08-03 09:28:56.000000000 -0500
@@ -333,14 +333,14 @@
           sendmail-path, smtp-server, and compile-time options. The first
           MTA specified in the following list is used:
           
-         1. _sendmail-path_ in /usr/local/lib/pine.conf.fixed
-         2. _smtp-server_ in /usr/local/pine.conf.fixed
+         1. _sendmail-path_ in /etc/pine.conf.fixed
+         2. _smtp-server_ in /etc/pine.conf.fixed
          3. _sendmail-path_ specified on the command line.
          4. _smtp-server_ specified on the command line.
          5. _sendmail-path_ in the user's .pinerc file.
          6. _smtp-server_ in the user's .pinerc file.
-         7. _sendmail-path_ in /usr/local/lib/pine.conf
-         8. _smtp-server_ in /usr/local/pine.conf
+         7. _sendmail-path_ in /etc/pine.conf
+         8. _smtp-server_ in /etc/pine.conf
          9. DF_SENDMAIL_PATH defined at compile time.
         10. SENDMAIL and SENDMAILFLAGS defined at compile time.
             
@@ -670,14 +670,14 @@
    SYSTEM_PINERC
           The name of the file which holds _Pine_ configuration
           information for all users on the system. Default on UNIX
-          systems is /usr/local/lib/pine.conf.
+          systems is /etc/pine.conf.
    SYSTEM_PINERC_FIXED
           The name of the file which holds the same type of information
           as for SYSTEM_PINERC, but only for variables that the
           administrator wants to keep fixed. That is, users are not
           allowed to change variables that are specified in the FIXED
           file. Default on UNIX systems is
-          /usr/local/lib/pine.conf.fixed.
+          /etc/pine.conf.fixed.
           
    There are a couple of more obscure options which are in the source
    code because a few people have asked for them or because we changed
@@ -875,11 +875,10 @@
    the help text is compiled into _Pine_ so there are no _required_
    auxiliary files.
    
-   There are, however, three optional auxiliary files:
-   /usr/local/lib/pine.info, /usr/local/lib/pine.conf, and
-   /usr/local/lib/pine.conf.fixed. The file pine.info contains text on
-   how to get further help on the local system. It is part of the help
-   text for the main menu and should probably refer to the local help
+   There are, however, three optional auxiliary files: /etc/pine.info,
+   /etc/pine.conf, and /etc/pine.conf.fixed. The file pine.info contains
+   text on how to get further help on the local system. It is part of the
+   help text for the main menu and should probably refer to the local help
    desk or the system administrator. If this file doesn't exist a generic
    version which suggests ``talking to the computer support staff at your
    site'' is shown. The file pine.conf is used to set system-wide default
@@ -976,11 +975,11 @@
    This section lists the various files which _Pine_ uses which are not
    email folders. All of these are the default names of files, they may
    vary based on _Pine_'s configuration.
-   /usr/local/lib/pine.conf
+   /etc/pine.conf
           Pine's global configuration file.
-   /usr/local/lib/pine.conf.fixed
+   /etc/pine.conf.fixed
           Non-overridable global configuration file.
-   /usr/local/lib/pine.info
+   /etc/pine.info
           Local pointer to system administrator.
    ~/.pinerc
           Personal configuration file for each user.
@@ -1200,10 +1199,10 @@
           Configuration: Prints a sample system configuration file to the
           screen or standard output. To generate an initial system
           configuration file, execute
-                pine -conf > /usr/local/lib/pine.conf
+                pine -conf > /etc/pine.conf
           To generate a system configuration file using settings from an
           old system configuration file, execute
-                pine -P old-pine.conf -conf > /usr/local/lib/pine.conf
+                pine -P old-pine.conf -conf > /etc/pine.conf
           A system configuration file is not required.
    -convert_sigs _-p pinerc_
           Convert signatures contained in signature files into literal
@@ -1313,7 +1312,7 @@
           configuration folder.
    -P _pinerc_
           Uses the named file as the system wide configuration file
-          instead of _/usr/local/lib/pine.conf_ on UNIX, or nothing on
+          instead of _/etc/pine.conf_ on UNIX, or nothing on
           _PC-Pine_. Pinerc may be either a local file or a remote
           configuration folder.
    -passfile _passfile_
@@ -1519,8 +1518,8 @@
    configuration. In most cases, the compiled-in preferences will suit
    users and administrators just fine. When running _Pine_ on a UNIX
    system, the default built-in configuration can be changed by setting
-   variables in the system configuration files, /usr/local/lib/pine.conf
-   or /usr/local/lib/pine.conf.fixed. (Actually, these files are whatever
+   variables in the system configuration files, /etc/pine.conf
+   or /etc/pine.conf.fixed. (Actually, these files are whatever
    the definitions for SYSTEM_PINERC and SYSTEM_PINERC_FIXED in
    pine/osdep/os-xxx.h are set to.) The location of the pine.conf file
    can be changed with the -P command line argument. Both _Pine_ and
@@ -3317,8 +3316,7 @@
           For Unix _Pine_ the program _ispell_ works well as an alternate
           spell checker. If your Unix system has _ispell_ it is probably
           reasonable to make it the default speller by configuring it as
-          the default in the system configuration file,
-          /usr/local/lib/pine.conf.
+          the default in the system configuration file, /etc/pine.conf.
           If this option is not set, then the system's _spell_ command is
           used. The spell command does not work the same as the alternate
           speller. It produces a list of misspelled words on its standard
@@ -3340,7 +3338,7 @@
           set to zero ssh connections will be completely disabled.
    _ssh-path_
           Sets the name of the command used to open a UNIX secure shell
-          connection. The default is typically /usr/local/bin/ssh.
+          connection. The default is typically /usr/bin/ssh.
    _standard-printer_
           System-wide configuration file only. Specifies a list of
           commands for category 2 of the _Setup/Printer_ screen, the
@@ -8385,10 +8383,10 @@
     5. the system-wide _fixed_ configuration file (Unix _Pine_ only)
        
    The fixed configuration file is normally
-   /usr/local/lib/pine.conf.fixed.
+   /etc/pine.conf.fixed.
    
    The system-wide configuration file is normally
-   /usr/local/lib/pine.conf for Unix _Pine_ and is normally not set for
+   /etc/pine.conf for Unix _Pine_ and is normally not set for
    _PC-Pine_. For _PC-Pine_, if the environment variable _$PINECONF_ is
    set, that is used for the system-wide configuration. This location can
    be set or changed on the command line with the -P flag. The
@@ -8633,8 +8631,8 @@
      * A program that implements the SMTP or ESMTP protocol via stdio.
      * An entry in /etc/services for the alternate service.
      * An entry in /etc/inetd.conf for the alternate service.
-     * An entry in /usr/local/lib/pine.conf,
-       /usr/local/lib/pine.conf.fixed or ~/.pinerc.
+     * An entry in /etc/pine.conf,
+       /etc/pine.conf.fixed or ~/.pinerc.
      _________________________________________________________________
    
 MIME.Types file
@@ -9818,7 +9816,7 @@
    
    The second selection is the standard UNIX print command. The default
    is _lpr_, but it can be changed on a system basis to anything so
-   desired in /usr/local/lib/pine.conf.
+   desired in /etc/pine.conf.
    
    The third selection is the user's personal choice for a UNIX print
    command. The text to be printed is piped into the command. _Enscript_
@@ -10272,9 +10270,9 @@
           Error messages for incorrect terminal types (try "foo" and
           "vt52")
    ___
-          Reading of /usr/local/lib/pine.conf
+          Reading of /etc/pine.conf
    ___
-          Fixing variables and features in /usr/local/lib/pine.conf.fixed
+          Fixing variables and features in /etc/pine.conf.fixed
    ___
           Flag command (check message status changed in mail folder)
    ___
diff -ur pine4.56/pine/init.c pine4.56.fhs/pine/init.c
--- pine4.56/pine/init.c	2003-05-23 11:14:31.000000000 -0500
+++ pine4.56.fhs/pine/init.c	2003-08-03 09:28:56.000000000 -0500
@@ -360,7 +360,7 @@
 
 CONF_TXT_T cf_text_ssh_open_timeo[] =	"Sets the time in seconds that Pine will attempt to open a UNIX secure\n# shell connection.  The default is 15, min is 5, and max is unlimited.\n# Zero disables ssh altogether.";
 
-CONF_TXT_T cf_text_ssh_path[] =		"Sets the name of the command used to open a UNIX secure shell connection.\n# Tyically this is /usr/local/bin/ssh.";
+CONF_TXT_T cf_text_ssh_path[] =		"Sets the name of the command used to open a UNIX secure shell connection.\n# Tyically this is /usr/bin/ssh.";
 
 CONF_TXT_T cf_text_ssh_command[] =	"Sets the format of the command used to open a UNIX secure\n# shell connection.  The default is \"%s %s -l %s exec /etc/r%sd\"\n# NOTE: the 4 (four) \"%s\" entries MUST exist in the provided command\n# where the first is for the command's path, the second is for the\n# host to connnect to, the third is for the user to connect as, and the\n# fourth is for the connection method (typically \"imap\")";
 
@@ -402,7 +402,7 @@
 
 /*----------------------------------------------------------------------
 These are the variables that control a number of pine functions.  They
-come out of the .pinerc and the /usr/local/lib/pine.conf files.  Some can
+come out of the .pinerc and the /etc/pine.conf files.  Some can
 be set by the user while in Pine.  Eventually all the local ones should
 be so and maybe the global ones too.
 
diff -ur pine4.56/pine/osdep/os-lnx.h pine4.56.fhs/pine/osdep/os-lnx.h
--- pine4.56/pine/osdep/os-lnx.h	2003-05-23 11:07:00.000000000 -0500
+++ pine4.56.fhs/pine/osdep/os-lnx.h	2003-08-03 09:28:56.000000000 -0500
@@ -60,7 +60,7 @@
    there should be a dummy disk_quota() routine in os-xxx.c so that defining
    this won't be harmful anyway.
  ----*/
-/* #define USE_QUOTAS */
+#define USE_QUOTAS
 
 
 
@@ -74,7 +74,7 @@
    default-composer-hdrs or customized-hdrs to get at it. Instead of
    defining NEVER_ALLOW_CHANGING_FROM, an easier way of preventing From
    changing is to put the feature "no-allow-changing-from" in the
-   /usr/local/lib/pine.conf.fixed file.
+   /etc/pine.conf.fixed file.
  ----*/
 /* #define NEVER_ALLOW_CHANGING_FROM */
 
diff -ur pine4.56/pine/pine.hlp pine4.56.fhs/pine/pine.hlp
--- pine4.56/pine/pine.hlp	2003-05-29 13:10:13.000000000 -0500
+++ pine4.56.fhs/pine/pine.hlp	2003-08-03 09:28:56.000000000 -0500
@@ -969,9 +969,9 @@
  executable     &lt;Unix search path&gt;/pine
  persnl cfg     ~/.pinerc
  except cfg     ~/.pinercex
- global cfg     /usr/local/lib/pine.conf
- fixed cfg      /usr/local/lib/pine.conf.fixed
- local help     /usr/local/lib/pine.info
+ global cfg     /etc/pine.conf
+ fixed cfg      /etc/pine.conf.fixed
+ local help     /etc/pine.info
 
  interrupted    ~/.pine-interrupted-mail
  debug          ~/.pine-debugN
@@ -1409,9 +1409,9 @@
 this link now to send a message to it:<BR>
 <A HREF="X-Pine-Gripe:_LOCAL_ADDRESS_?local"><!--#echo var="_LOCAL_FULLNAME_"--></A><P>
 <!--chtml endif-->
-<!--chtml if [ -r /usr/local/lib/pine.info ]-->
+<!--chtml if [ -r /etc/pine.info ]-->
 <HR WIDTH="75%">Local Support Contacts:<P>
-<!--#include file="/usr/local/lib/pine.info"-->
+<!--#include file="/etc/pine.info"-->
 <HR WIDTH="75%">
 <!--chtml endif-->
 <P> 
@@ -1499,7 +1499,7 @@
 <OL>
   <LI> <A HREF="#intro">Introduction</A>
   <LI> <A HREF="#pine_help">Pine Help</A>
-<!--chtml if [ -r /usr/local/lib/pine.info ]-->
+<!--chtml if [ -r /etc/pine.info ]-->
   <LI> <A HREF="#local_support">Local Support Contacts</A>
 <!--chtml endif-->
   <LI> <A HREF="#giving_cmds">Giving Commands in Pine</A>
@@ -1611,9 +1611,9 @@
 at your site.)
 
 
-<!--chtml if [ -r /usr/local/lib/pine.info ]-->
+<!--chtml if [ -r /etc/pine.info ]-->
 <H2><A NAME="local_support">Local Support Contacts</A></H2>
-<!--#include file="/usr/local/lib/pine.info"-->
+<!--#include file="/etc/pine.info"-->
 <!--chtml endif-->
 
 <H2><A NAME="giving_cmds">Giving Commands in Pine</A></H2>
@@ -2299,7 +2299,7 @@
 file, execute
 
 <PRE><CODE>
-		pine -conf > /usr/local/lib/pine.conf
+		pine -conf > /etc/pine.conf
 </CODE></PRE>
 <P>
 
@@ -2307,7 +2307,7 @@
 system configuration file, execute
 
 <PRE><CODE>
-		pine -P old-pine.conf -conf > /usr/local/lib/pine.conf
+		pine -P old-pine.conf -conf > /etc/pine.conf
 </CODE></PRE>
 <P>
 A system configuration file is not required.
@@ -2472,7 +2472,7 @@
 <DT> -P <EM>pinerc</EM>
 
 <DD> Uses the named file as the system wide configuration file instead of
-<EM>/usr/local/lib/pine.conf</EM> on UNIX, or nothing on <EM>PC-Pine</EM>.
+<EM>/etc/pine.conf</EM> on UNIX, or nothing on <EM>PC-Pine</EM>.
 Pinerc may be either a local file or a remote configuration folder.
 <P>
 
@@ -3200,10 +3200,10 @@
 </OL>
 <P>
 The fixed configuration file is normally
-<CODE>/usr/local/lib/pine.conf.fixed</CODE>.
+<CODE>/etc/pine.conf.fixed</CODE>.
 <P>
 The system-wide configuration file is normally
-<CODE>/usr/local/lib/pine.conf</CODE> for Unix Pine and is normally not
+<CODE>/etc/pine.conf</CODE> for Unix Pine and is normally not
 set for PC-Pine.
 For PC-Pine, if the environment variable <EM>$PINECONF</EM> is set, that
 is used for the system-wide configuration.
@@ -19021,7 +19021,7 @@
 <H1>OPTION: ssh-path</H1>
 
 Sets the name of the command used to open a UNIX secure shell
-connection. The default is typically <CODE>/usr/local/bin/ssh</CODE>.
+connection. The default is typically <CODE>/usr/bin/ssh</CODE>.
 <P>
 <UL>   
 <LI><A HREF="h_finding_help">Finding more information and requesting help</A>
